.github/workflows/ci.yml

```diff
@@ -21,70 +22,87 @@ jobs:
       RUSTFLAGS: -D warnings
 
     steps:
-      - name: ☑️ Clone repository
+      - name: Clone repository
         uses: actions/checkout@v3
 
-      - name: ➡️ Install rust
+      - name: Install rust
         uses: dsherret/rust-toolchain-file@v1
 
-      - name: 🗃️ Cache
+      - name: Cache
         uses: Swatinem/rust-cache@v1
 
-      - name: ➡️ Install up Deno
+      - name: Install up Deno
         uses: denoland/setup-deno@v1.0.0
 
-      - name: 💄 Format
+      - name: Format
         if: contains(matrix.os, 'ubuntu')
         run: |
           cargo fmt -- --check
           deno fmt --check
 
-      - name: 💄 Lint
+      - name: Lint
         if: contains(matrix.os, 'ubuntu')
         run: |
           cargo clippy --locked --release --all-features --all-targets -- -D clippy::all
           deno lint
 
-      - name: 🏗️ Build
+      - name: Cargo Build
         run: cargo build --locked --release --all-features --all-targets
 
-      - name: 🧪 Test
-        run: |
-          cargo test --locked --release --all-features --all-targets
-          deno task test
+      - name: Cargo Test
+        run: cargo test --locked --release --all-features --all-targets
 
-      - name: 💄 Ensure Wasm up to date
-        if: contains(matrix.os, 'ubuntu')
-        run: deno task build --check
+      - name: Build Wasm
+        run: deno task build
+
+      - name: Test JavaScript
+        run: deno task test
 
-      - name: 📦 Publish
+      - name: Cargo publish
         if: |
           contains(matrix.os, 'ubuntu') &&
           github.repository == 'denoland/deno_graph' &&
           startsWith(github.ref, 'refs/tags/')
         env:
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-        run: |
-          cargo publish -vv
+        run: cargo publish
 
-        # NPM Distribution
-      - name: ➡️ get tag version
+      - name: Get tag version
         if: contains(matrix.os, 'ubuntu') && startsWith(github.ref, 'refs/tags/')
         id: get_tag_version
         run: echo ::set-output name=TAG_VERSION::${GITHUB_REF/refs\/tags\//}
 
-      - name: ➡️ setup node
+      - name: Build deno.land/x
+        uses: s0/git-publish-subdir-action@92faf786f11dfa44fc366ac3eb274d193ca1af7e
```

Comment:
This seems funky, could you explain how it works?

Reply:
It takes a subdirectory, in this case "js", and pushes it to the "deno_registry" branch (https://github.com/denoland/deno_graph/tree/deno_registry).

https://github.com/s0/git-publish-subdir-action#github-action-push-git-subdirectory-as-branch